code/modules/surgery/organs/stomach/external/wings.dm

```diff
@@ -0,0 +1,226 @@
+///Wing base type. doesn't really do anything
+/obj/item/organ/external/wings
+	zone = BODY_ZONE_CHEST
+	slot = ORGAN_SLOT_EXTERNAL_WINGS
+	layers = EXTERNAL_BEHIND | EXTERNAL_ADJACENT | EXTERNAL_FRONT
+
+	preference = "wings"
+
+/obj/item/organ/external/wings/can_draw_on_bodypart(mob/living/carbon/human/human)
+	if(!(human.wear_suit?.flags_inv & HIDEJUMPSUIT))
```

Comment:
Rewrite this as if this is the opposite, early-return FALSE. That way we dont have two ifs in each other where an `&&` looks more correct.